File: playground/devfancy/commerce/src/main/java/team/sipe/commerce/domain/Payment.java
package team.sipe.commerce.domain;

import team.sipe.commerce.common.BaseEntity;
import lombok.Getter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Getter
@Table(name = "PAYMENTS")
@Entity
public class Payment extends BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String paymentMethod; // 결제 수단 ex) BILLING_NAVER_PAY
    private String paymentMethodName; // 결제 수단 이름 ex) 네이버
    private int paymentAmount; // 총 결제 금액

    protected Payment() {
    }

    public Payment(final String paymentMethod, final String paymentMethodName, final int paymentAmount) {
        this.paymentMethod = paymentMethod;
        this.paymentMethodName = paymentMethodName;
        this.paymentAmount = paymentAmount;
    }
}

File: playground/ruthetum/app/e-commerce/order/src/main/kotlin/team/sipe/spurt/ecommerce/order/domain/Payment.kt
package team.sipe.spurt.ecommerce.order.domain

import jakarta.persistence.*
import org.hibernate.annotations.Comment

@Table(name = "payments")
@Entity
class Payment(
    @Comment("결제 수단")
    @Column(name = "payment_method")
    val paymentMethod: String,

    @Comment("결제 수단 이름")
    @Column(name = "payment_method_name")
    val paymentMethodName: String,

    @Comment("총 결제 금액")
    @Column(name = "payment_amount")
    val paymentAmount: Int,

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,
) : team.sipe.spurt.ecommerce.order.common.BaseEntity()